### PR TITLE
Added list-w command

### PR DESCRIPTION
**Pull request by LiuXuanIan**
Added list-w command to list all the worklists, and add some test files to test that command.

### CODE REVIEW

File: src/test/java/seedu/address/model/equipment/UniqueWorkListListTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+import seedu.address.model.equipment.exceptions.DuplicateEquipmentException;
+import seedu.address.model.equipment.exceptions.EquipmentNotFoundException;
+//import seedu.address.testutil.EquipmentBuilder;
+
+public class UniqueWorkListListTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private final WorkList typicalWorkListA = new WorkList("2019-01-01", "Alice");
```
Comment (e0191632) [inline src/test/java/seedu/address/model/equipment/UniqueWorkListListTest.java]: Shouldn't have Alice anymore. Alice is changed to Anchorvale CC name
I havent changed bob so it should still be fine.

File: src/test/java/seedu/address/testutil/TypicalWorkLists.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import seedu.address.model.EquipmentManager;
+//import seedu.address.model.equipment.Equipment;
+import seedu.address.model.equipment.WorkList;
+
+/**
+ * A utility class containing a list of {@code WorkList} objects to be used in tests.
+ */
+public class TypicalWorkLists {
+
+    public static final WorkList LISTA = new WorkList("2019-01-01", "ALICE");
```
Comment (e0191632) [inline src/test/java/seedu/address/testutil/TypicalWorkLists.java]: same for this line. alice change to achorvale CC

Comment (LiuXuanIan) [reply src/test/java/seedu/address/testutil/TypicalWorkLists.java]: Okay. I will change it right now. Thank you Rachel for your comments.

**Comment by e0191632** [comment]
looks good! 💃

**Pull request merged**